sprint-mission5/public/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
```

Comment:
```suggestion
<html lang="ko">
<head>
  <meta charset="UTF-8">
</head>
```

html lang="ko" 값으로 변경 되어야합니다. 😸

```
- 언어 하위태그(필수)
두 세 글자로 구성된 코드로 기본 언어를 정의하며, 보통 모두 소문자로 표기합니다. 예를 들어 영어의 언어 태그는 en이고, 한국어의 언어 태그는 ko입니다.
```

https://developer.mozilla.org/ko/docs/Web/HTML/Global_attributes/lang